datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -127,7 +126,7 @@ public SdxClusterResponse getByCrn(@ResourceCrn String clusterCrn) {
     }
 
     @Override
-    @FilterListBasedOnPermissions(action = AuthorizationResourceAction.DESCRIBE_DATALAKE)
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.LIST_DATALAKES)
     public List<SdxClusterResponse> list(String envName) {
```

Comment:
Just a note. The list datahub page shows the create datahub button if this list sdx endpoint gives back some datalakes. So after this change a user will see this button enabled and later on will figure out he doesn't have right to create datahub beacuse there is no datalake that he can use.

Reply:
Yes, this is the same as we can click on a datalake and later we may deny to describe it. I think we already discussed  disadvantages of this solution.
Anyway authorization of buttons and UI elements is ahead of us I think.

Reply:
DatahubCreator needs be able to "list" the environments in a particular environment with it's environments/listDatalakes or environemnts/describeDatalake. Since we assign this on environment level it would be easy to solve.